packages/fluent-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -59,49 +53,35 @@ const SelectWidget = ({
 }: WidgetProps) => {
   const { enumOptions, enumDisabled } = options;
 
-  const emptyValue = multiple ? [] : '';
+  const _onChange = ({}, value: any) => onChange(value.key);
 
-  const _onChange = ({
-    target: { value },
-  }: React.ChangeEvent<{ name?: string; value: unknown }>) =>
-    onChange(processValue(schema, value));
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, processValue(schema, value));
   const _onFocus = ({
     target: { value },
   }: React.FocusEvent<HTMLInputElement>) =>
     onFocus(id, processValue(schema, value));
 
+  const dropDown = (enumOptions as any[]) || [];
+
+  const newOptions = dropDown.map(option => ({
```

Comment:
Please remove the line above and say
```suggestion
  const newOptions = enumOptions.map(option => ({
```

Reply:
Done.